Makefile

```diff
@@ -145,6 +145,9 @@ else
 	$(error Please set GOOS=linux for building the image)
 endif
 
+clusterctl-image: generate fmt vet manifests
+	docker build . -f cmd/clusterctl/Dockerfile -t "$(REGISTRY)/clusterctl:$(VERSION)"
```

Comment:
k8scloudprovider/clusterctl:a5ecff51-dirty will be name for this 

I suggest to rename this to openstack-cluster-api-clusterctl to avoid confusion..
https://hub.docker.com/search/?q=k8scloudprovider&type=image

Reply:
Thanks @jichenjc , +1 to rename for this as different cloud providers will have different clusterctl.

/lgtm cancel

Reply:
thanks for comments, updated.